docs/working-with-js-objects-and-values.md

```diff
@@ -188,7 +202,7 @@ manual](https://v2.ocaml.org/manual/objectexamples.html).
 We have already explored one approach for creating JavaScript object literals by
 using [`Js.t` values and the `mel.obj` extension](#using-js-t-objects).
 
-Melange additionally offers the `mel.obj` attribute, which can be used in
+Melange additionally offers the `@mel.obj` attribute, which can be used in
```

Comment:
The problem with adding `@` here is that OCaml syntax uses two `@@` and Reason one `@`. Before it was syntax agnostic, but now it's not. Maybe we can use the "inline syntax switching" elements to work around this.

Reply:
Makes sense, fixed in another PR: https://github.com/melange-re/melange-re.github.io/pull/212/commits/ba9bf1d433289f0823397bc4db0c128f02e8d6c3

I was used to referring to annotations with `@` prefixed to the name because the book only uses Reason syntax, so there's no ambiguity.